ambry-store/src/main/java/com.github.ambry.store/Log.java

```diff
@@ -173,6 +181,20 @@ long getSegmentCapacity() {
   }
 
   /**
+   * @return the number of unallocated segments for this log.
```

Comment:
can you add a caveat here that this can only be used when compaction is not running?

Reply:
added